Ugly_Number/Ugly_Number.py

```diff
@@ -0,0 +1,21 @@
+# Ugly numbers are numbers whose only prime factors are 2, 3 or 5.
+x = False
```

Comment:
Blank line

Reply:
where ?

Reply:
do we need to give a blank line after comments too?

Reply:
It is the start of code, one line gap after description provides a clear view of where actual code starts